src/redux/modules/spinner/spinner.js

```diff
@@ -0,0 +1,97 @@
+/* @flow */
+import debug from 'debug';
+import {
+  UPDATE_USER_REQUEST,
+  UPDATE_USER_SUCCESS,
+  UPDATE_USER_FAILURE,
+} from 'redux/modules/user/user-actions';
+import {
+  LOGIN_REQUEST,
+  LOGIN_SUCCESS,
+  LOGIN_FAILURE,
+  LOGOUT_REQUEST,
+  LOGOUT_SUCCESS,
+} from 'redux/modules/auth/auth-actions';
```

Comment:
Isn't it easier to `dispatch(showSpinner(message))`?
With this approach you handle all request, success, and failures, for _every_ async action here. Leaves two places to refactor when you refactor redux modules.